src/biome/text/commands/explore/explore.py

```diff
@@ -7,7 +7,7 @@
 from allennlp.commands.subcommand import Subcommand
 from allennlp.common.checks import ConfigurationError
 from biome.data.sources import DataSource
-from biome.data.utils import ENV_ES_HOSTS
+from ...environment import ES_HOST, BIOME_EXPLORE_ENDPOINT
```

Comment:
I think in general absolute imports are preferred over relative ones except for a few cases (https://realpython.com/absolute-vs-relative-python-imports/)